app/account/page.jsx

```diff
@@ -0,0 +1,109 @@
+'use client'
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import { useEffect, useState } from 'react'
+import { useUserContext } from '../context/usercontext'
+import { useRouter } from 'next/navigation'
+import Profile from '../components/profile'
+
+function CustomTabPanel(props) {
+  const { children, value, index, ...other } = props;
+
+  
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`simple-tabpanel-${index}`}
+      aria-labelledby={`simple-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box sx={{ p: 3 }}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+CustomTabPanel.propTypes = {
+  children: PropTypes.node,
+  index: PropTypes.number.isRequired,
+  value: PropTypes.number.isRequired,
+};
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`,
+  };
+}
+
+export default function BasicTabs() {
+    const [value, setValue] = React.useState(0);
+    
+    const {user, setUser, userData, setUserData} = useUserContext();
+    const [dataLoaded, setDataLoaded] = useState(false);
+    let prevUser = null;
+    const router = useRouter();
+
+    useEffect(()=>{
+        if (user===null) {
+        try {
+            prevUser = JSON.parse(sessionStorage.getItem('firebase:authUser:AIzaSyCoGURJeUWdIylWkAEDYEpOqY6YnAaJYy0:[DEFAULT]'))
```

Comment:
do you need to use this key in multiple places? It might be better to have this hard coded in one location like a `constants` file and then import it wherever you need to use it. This way if you ever need to change this key you only have to change it in one location instead of multiple. 
This same principle typically applies to most other hard coded strings in your repository.